scripts/handler.js

```diff
@@ -0,0 +1,29 @@
+const schedule = require('node-schedule');
+const jobsModel = require('../app/models/jobsModel.js');
+const clientsModel = require('../app/models/clientsModel.js');
+const mailer = require('../modules/mailer.js');
+
+/**
+ * Adds scheduled jobs for sending emails by checking for
+ * classes starting in 30 min every 15 min using chron job 
+ * and jobsModel script. Loop through each class to get clients
+ * and then loop through clients to get their emails. Finally,
+ * send the email to all clients of each class.
+ */
+async function scheduleEmail() {
+    schedule.scheduleJob('*/15 * * * *', async function() {
+        const classArr = await jobsModel.getByMinutesFromNow(30);
```

Comment:
Utilizing Join, you can combine the function of getByMinutesFromNow as well as the client email into one function. When you are querying from the database, use join to join the jobs table as well as the clients table

Reply:
Added join usage